docs/src/main/sphinx/release/release-454.md

```diff
@@ -0,0 +1,88 @@
+# Release 454 (7 Aug 2024)
+
+## General
+
+* Add support for retrieving lightweight query information from the `QueryInfo`
+  endpoint with the new `pruned` property. ({issue}`20843`)
+* Improve performance of queries with multiple distinct aggregations. ({issue}`22355`)
+* Skip query completed notifications in the event listener when there are too
+  many concurrent completion requests. The maximum number of requests to handle
+  before skipping can be configured with the
+  `event-listener.max-concurrent-query-completed-events` configuration property. ({issue}`22911`)
+* Fix failure for queries involving {func}`json_parse` and a cast to `array`,
+  `map`, or `row`. ({issue}`22840`)
+
+## Security
+
+* Fix the Trino user incorrectly defaulting to the `user.name` property when no
```

Comment:
Is this for the JDBC driver? CLI? It should be moved to the appropriate section(s).

Also, the phrase "`user.name` property" doesn't mean anything for a user of the CLI or JDBC driver. Is it the user running the process? cc @wendigo

Reply:
yes, that's the user running the process and this is for JDBC driver